deploy-argocd.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash -e
+
+# Deploy Applications to ArgoCD on EKS Cluster
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+main() {
+    verify_permissions || exit $?
+    login_argocd
+    test_pvc_binding
+    deploy_apps
+}
+
+verify_permissions() {
+    if [ "$(kubectl auth can-i '*' '*' --all-namespaces)" != "yes" ]; then
+        echo
+        echo "[ERROR] User '$(oc whoami)' does not have the required 'cluster-admin' role." 1>&2
+        echo "Log into the cluster with a user with the required privileges (e.g. kubeadmin) and retry."
+        return 1
+    fi
+}
+
+retry() {
+    for _ in {1..3}; do
+        local ret=0
+        $1 || ret="$?"
+        if [[ "$ret" -eq 0 ]]; then
+            return 0
+        fi
+        sleep 3
+    done
+
+    echo "$1": "$2."
+    return "$ret"
+}
+
+login_argocd() {
+    argocd login argocd.konflux-ci.net --sso
+}
+
+test_pvc_binding(){
+    local pvc_resources="${ROOT}/dependencies/pre-deployment-pvc-binding"
+    echo "Creating PVC from '$pvc_resources' using the cluster's default storage class"
+    kubectl apply -k "$pvc_resources"
+    echo "PVC binding successfull"
+}
+
+deploy_tekton() {
```

Comment:
If we are using ArogCD, why do we need this function ?

Reply:
I agre, I don't think we need this function.